zebrad/tests/common/lightwalletd/wallet_grpc_test.rs

```diff
@@ -1,17 +1,19 @@
-//! Test all grpc calls a wallet connected to a lightwalletd instance backed by zebra can do.
+//! Test all gRPC calls a wallet connected to a lightwalletd instance backed by
+//! zebra can do.
 //!
-//! This test requires a cached chain state that is partially synchronized, i.e., it should be a
-//! few blocks below the network chain tip height. It also requires a lightwalletd data dir in sync
-//! with the cached chain state.
+//! This test requires a cached chain state that is partially synchronized,
+//! i.e., it should be a few blocks below the network chain tip height. It also
+//! requires a lightwalletd data dir in sync with the cached chain state.
 //!
-//! Current coverage of all available rpc methods according to `CompactTxStreamer`:
+//! Current coverage of all available rpc methods according to
+//! `CompactTxStreamer`:
```

Comment:
Nitpick: For future PRs, can you configure your editor so it doesn't reflow paragraphs?

It's ok to change edited lines, but changing unrelated lines makes it hard to see which docs have actually changed.

(Feel free to just close this comment after you've read it!)

Reply:
Sure, thanks for the suggestion.